src/WebAppDIRAC/WebApp/handler/ResourceSummaryHandler.py

```diff
@@ -312,6 +310,8 @@ def web_action(self, action=None, **kwargs):
 
     def web_getResourceSummaryData(self, name=None, status=None, elementType=None, statusType=None, tokenOwner=None):
         """This method returns the data required to fill the grid."""
+        if isinstance(elementType, str):  # it is passed in as e.g. '["Catalog"]' 
+            elementType = eval(elementType)
```

Comment:
```suggestion
            elementType = [ele.strip("\"' ") for ele in elementType.strip("[]").split(",")]
```

Reply:
Is this being passed as `json`? If so it should be `json.loads(elementType)` else failing that `ast.literal_eval` would be a better choice.

Reply:
Updated to use `json` (seems like it's done elsewhere too)